app/src/common/utils/attributeUtils.js

```diff
@@ -18,6 +18,9 @@ export const getAttributeValue = ({ key, value } = {}) => value || key;
 
 export const formatAttribute = ({ key, value } = {}) => {
   if (key && value) {
+    if (key.includes(':')) {
```

Comment:
If the both `key` and `value` presented the `key` must not contain trailing `:`.